syntax = "proto3";

message SmState {
  bool switch = 1;
  bool on = 2;
  bool direction = 3;
}

message SmParams {
  uint64 timeout = 1;
}